docs/docs/reference/crd-reference/app.md

```diff
@@ -165,14 +133,36 @@ spec:
 
 ## Differences between versions
 
-- The `spec.revision` field is introduced in v1alpha2.
-- The pre/post-deployment tasks and evaluations are not
-supported anymore in v1beta1.
+The synopsis of the `KeptnApp` resource
+is changed in `v1beta1` library:
+
+- The `apiVersion` field now references the `v1beta1` version
+  of the `lifecycle.keptn.sh` API.
+- `KeptnApp` must be
+  [autogenerated](../../guides/auto-app-discovery.md);
+  you cannot create it manually by applying a manifest.
+- The `name` and `namespace` fields are populated
+  from the fields of the same name in the `KeptnAppContext` resource
+  and must have identical values to those
+  in the corresponding `KeptnAppContext` resource.
+- The `app.kubernetes.io/managed-by: keptn` label or annotation
```

Comment:
it's not completely new, it's added always when the KeptnApp is auto-generated, so it was there also for v1alpha3 when using the auto-app discovery feature

Reply:
Fixed